example/main.go

```diff
@@ -0,0 +1,115 @@
+// This is an example server demonstrating aibridge usage.
+// Run with: go run ./example
```

Comment:
Does `go run ./example` work? Shouldn't it be `go run github.com/coder/aibridge/example` or `go run .`?